biocrnpyler/__init__.py

```diff
@@ -15,6 +15,8 @@
 from .dna_part_terminator import *
 from .construct_explorer import *
 from .integrase_enumerator import *
+from .components_combinatorial_complex import *
+from .components_combinatorial_conformation import *
 
```

Comment:
I have a general concern here, not directly related to this PR. Should we be importing all classes and functions in `__init__.py`? Until now we have been importing each and everything here but now I am wondering if this is a good software engineering practice. Anyone has any thoughts on this @zoltuz @WilliamIX @dr3y ?

Reply:
I've thought about this too....I think it might be good (but a pain in the ass) to refactor the biocrnpyler file structure to something like:

- BioCRNpyler
- - Mixture
- - - mixture.py, mixtures_cell.py etc.
- - Component
- - - component.py, components_basic.py etc
- - Mechanisms
etc

I think having "From biocrnpyler import *" grab all of these is then great, but that also gives easy leeway to do things like from biocrnpyler.mixture import Mixture if we want to be more careful about it.

Lets not make that part of this PR though.